proto/validate/validate.proto

```
@@ -0,0 +1,862 @@
+syntax = "proto2";
+package validate;
```

Buf (1.47.2)

Files with package "validate" must be within a directory "validate" relative to root but were in directory "proto/validate".

(PACKAGE_DIRECTORY_MATCH)